tests/resolver_t.py

```diff
@@ -293,19 +294,10 @@ def test_simple_http_resolver(self):
         #Test with a full uri
         #Note: This seems weird but idents resolve wrong and removes a slash from //
         ident = quote_plus('http://sample.sample/0001')
-        expected_path = join(self.app.resolver.cache_root, 'http')
-        expected_path = join(expected_path, '32')
-        expected_path = join(expected_path, '3a5')
-        expected_path = join(expected_path, '353')
-        expected_path = join(expected_path, '8f5')
-        expected_path = join(expected_path, '0de')
-        expected_path = join(expected_path, '1d3')
-        expected_path = join(expected_path, '011')
-        expected_path = join(expected_path, '675')
-        expected_path = join(expected_path, 'ebc')
-        expected_path = join(expected_path, 'c75')
-        expected_path = join(expected_path, '083')
-        expected_path = join(expected_path, 'loris_cache.tif')
+        expected_path = os.path.join(
+            self.app.resolver.cache_root, 'http',
+            'aa/003/aa5/c26/7b8/633/39a/d76/cd0/30d/c3c', 'loris_cache.tif'
+        )
```

Comment:
why is this expected path different now? Shouldn't the cache paths still resolve to the same place they used to?

Reply:
Eesh, looks like I missed an unquote somewhere. Will fix.